test/e2e/alertmanager_test.go

```diff
@@ -28,86 +28,89 @@ import (
 )
 
 func TestAlertmanagerCreateDeleteCluster(t *testing.T) {
+	t.Parallel()
+
+	ctx := testFramework.NewTestCtx(t)
+	defer ctx.CleanUp(t)
+	ctx.BasicSetup(t, framework.KubeClient)
```

Comment:
Ah, the Go convention is to always keep abbreviations uppercase, e.g. `ServeHTTP`. So `ctx.Id` should be `ctx.ID`.